examples/sampling/graphbolt/rgcn/hetero_rgcn.py

```diff
@@ -127,6 +126,8 @@ def create_dataloader(
     # Move the mini-batch to the appropriate device.
     # `device`:
     #   The device to move the mini-batch to.
+    # [TODO] GPU sampling is not supported yet.
```

Comment:
The TODO is not accurate, we need this hack is because to minibatch does not support to GPU, we need to convert it to dgl block first.